### ISSUE TITLE
Results get worse as training increases

### ISSUE THREAD

**Issue by coded5282** [junyanz/pytorch-CycleGAN-and-pix2pix#1032]
For some reason, when I train the CycleGAN the images on the left lose their edges/contrast after being converted to images with similar style to those on the right as I train for more epochs. So basically, epoch 0 seems to be producing the best results - it gets worse with more training. 

Is this a sign of overfitting or something else? Would you happen to have any suggestions for how to remedy this?

For context, I'm using roughly ~3k images in both domain A and domain B. I'm not using any identity loss and discriminator loss is halved.

Thank you very much!

**Comment by junyanz** [comment]
Not sure. Sometimes, the source and target domain may have different object statistics. To prevent bigger changes, you can add an identity loss.
